helmfile/charts/gatekeeper-constraints/values.yaml

```diff
@@ -7,5 +7,10 @@ requireNetworkpolicies:
 requireResourceRequests:
     enable: true
     enforcementAction: dryrun
+restrictLatestTag:
+    enable: true
+    enforcementAction: dryrun
 
 imageRegistryURL: registry.example.com
+
+imageRestrictedTag: latest
```

Comment:
Thanks for the feedback !! Have done the changes . :)

Reply:
I think you still missed to update this line, it should also default to a list and not a string. And also I would like to rename it to `imageRestrictedTags` since it can be multiple

Reply:
Thanks Fredrik !! I have fixed now